src/test/java/com/glovoapp/planningpoker/e2e/nametest/SpecialCharacterTest.java

```diff
@@ -0,0 +1,41 @@
+package com.glovoapp.planningpoker.e2e.nametest;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SpecialCharacterTest {
+
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private SessionPage sessionPage = new SessionPage(driver);
+
+    private final String userName = "SOMETEXT:";
```

Comment:
Constant should be `static`:

```suggestion
    private static final String USERNAME = "SOMETEXT:";
```